src/data-dictionary/IgSummary.ts

```diff
@@ -414,7 +451,7 @@ export class IgSummary {
             size: 16
         };
         [
-            ['', 'IG name', this.sushiConfig.title],
+            ['', 'IG name', this.sushiConfig.name],
```

Comment:
is this an issue with the specific igs this feature is used on, or is it a general fix needed to the ig-summary code?

Reply:
I believe name is essential since it's from IG: https://fshschool.org/docs/sushi/configuration/#fsh-only-existing-implementationguide-resource and title is kinda recommended : https://fshschool.org/docs/sushi/configuration/#fsh-and-ig-processing-recommended-configuration

Based on my experience, not many IGs have sushiconfig yaml that contains data for both. Often neither. So maybe we can put everything we can capture into the spreadsheet?

Reply:
including both seems reasonable